src/main/java/school/lemon/changerequest/java/extendedinteger/ExtendedInteger.java

```diff
@@ -1,5 +1,6 @@
 package school.lemon.changerequest.java.extendedinteger;
 
+import javax.naming.ldap.ExtendedRequest;
```

Comment:
Do you really need this import?

It's good practice to perform code formatting and imports optimization before making commit.